docs/api-reference/data-management-api.md

```diff
@@ -46,6 +47,22 @@ When you use these APIs concurrently with an indexing task or a kill task, the b
 Druid terminates some segments and marks others as used.
 Furthermore, it is possible that all segments could be unused, yet an indexing task might still be able to read data from these segments and complete successfully.
 
+### New Overlord APIs
```

Comment:
This section feels a little bit more like a release note or migration plan than a long lived doc section. In the big scheme of things, maybe it is not such a big deal as long as it is re-written when the coordinator apis are finally removed?

Reply:
Thanks for the suggestion, I have fixed this part up and added a release note in the PR description.